### PR TITLE
Update README.md with more HW and FW information

### PR DESCRIPTION
**Pull request by Hedda**
Update README.md for Zigbee Binding with more HW and FW information

### CODE REVIEW

File: org.openhab.binding.zigbee/README.md [hunk 1]
```diff
@@ -153,23 +153,23 @@ The following coordinators are known to be supported.
...
 
-Firmware versions higher than 6.4.0.0 may be required to work with some newer devices (i.e. those conforming to the Zigbee 3.0 standard). If your dongle has older firmware then devices may appear to join the network, but will not work as they will leave the network if they do not receive updated security information.
+Firmware versions higher than 6.4.0.0 may be required to work with some newer devices (i.e. those conforming to the Zigbee 3.0 standard). If your dongle has older firmware then devices may appear to join the network, but will not work as they will leave the network if they do not receive updated security information. Short summery of what versions is compatible with which hardware chips is that newer dongles are not compatible with older firmware just as older dongles usually not compatible with newer firmware.
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/README.md:172]: I'd suggest to remove this - for starters this makes no sense, but it's also not really relevant. For the most part it makes no difference from a firmware perspective - the only real difference relates to chipsets with 256k of memory or less which cannot run older firmware.

File: org.openhab.binding.zigbee/README.md [hunk 2]
```diff
@@ -183,6 +183,16 @@ If the USB dongle is not recognized, it might be necessary to make the dongle's
   ```
 - Plug in the dongle. It should now be recognized properly as ttyUSBx.
 
+##### EmberZNet and EZSP Protocol Version
+
+Silicon Labs (a.k.a. Silabs) Zigbee stack is called "EmberZNet PRO" (formerly "Ember Zigbee") and provides "EmberZNet Serial Protocol" (EZSP) as the protocol for a host application to interact with. openHAB is currenty support EZSP v5, v6, v7, and v8 which covers most if not all available Zigbee dongle based on Silabs Zigbee chips.
+
+Silicon Labs does not currently have a consolidated list of changes by EmberZNet SDK or EZSP protocol version. The EZSP additions, changes and deletions have only ever been listed in the "Zigbee EmberZNet Release Notes" (EmberZNet SDK) under the "New items" section as well as the Silabs UG100 EZSP Reference Guide included with each EmberZNet SDK release.
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/README.md:190]: Suggest to remove this as it's really not relevant to the user.

File: org.openhab.binding.zigbee/README.md [hunk 3]
```diff
@@ -183,6 +183,16 @@ If the USB dongle is not recognized, it might be necessary to make the dongle's
...
+
+Perhaps more important to know is that EZSP v5, v6 and v7 (EmberZNet 6.6.x.x) use the same framing format, while EZSP v8 (EmberZNet 6.7.x.x) introduced a new framing format and expanded command id field from 8 bits to 16 bits and is therefor not backward compatible.
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/README.md:192]: Again, suggest to remove as it's just not relevant to openHAB.

File: org.openhab.binding.zigbee/README.md [hunk 4]
```diff
@@ -183,6 +183,16 @@ If the USB dongle is not recognized, it might be necessary to make the dongle's
...
+
+Silicon Labs does not currently have a consolidated list of changes by EmberZNet SDK or EZSP protocol version. The EZSP additions, changes and deletions have only ever been listed in the "Zigbee EmberZNet Release Notes" (EmberZNet SDK) under the "New items" section as well as the Silabs UG100 EZSP Reference Guide included with each EmberZNet SDK release.
+
+Perhaps more important to know is that EZSP v5, v6 and v7 (EmberZNet 6.6.x.x) use the same framing format, while EZSP v8 (EmberZNet 6.7.x.x) introduced a new framing format and expanded command id field from 8 bits to 16 bits and is therefor not backward compatible. 
+
+The largest change was between EZSP v4 (first added in EmberZNet 4.7.2 SDK) and EZSP v5 that was added in EmberZNet 5.9.0 SDK which requires the Legacy Frame ID 0xFF. The change from EZSP v5 to EZSP v6 was done in EmberZNet 6.0.0 SDK. The change from EZSP v6 to EZSP v7 was in EmberZNet 6.4.0 SDK. EmberZNet 6.7.0 SDK added EZSP v8 (and Secure EZSP Protocol Version 2).
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/README.md:194]: Again, suggest to remove this as it's not relevant to users of the system.

It feels like you've copied this text from somewhere else?

**Review by cdjackson — CHANGES_REQUESTED** [review]
Thanks @Hedda . I'd suggest to remove some of this though as the technical detail about the protocol is really not relevant here.

**Comment by cdjackson** [comment]
@Hedda I just did a search on your proposed text and found that you have cut and paste this from another project which I'm definitely not comfortable with. Please read the contribution guidelines also and please sign the commit to say that you wrote this text or have the right to it.

**Comment by Hedda** [comment]
@cdjackson  FYI, yes I copied the text from a other project but it what I who wrote that text for that project:

https://github.com/zigpy/zigpy/commits?author=Hedda

File: org.openhab.binding.zigbee/README.md (hunk 1 above)
Comment (Hedda) [reply org.openhab.binding.zigbee/README.md]: Changed it moving the small part I still think is relative from and removing the rest.

File: org.openhab.binding.zigbee/README.md (hunk 3 above)
Comment (Hedda) [reply org.openhab.binding.zigbee/README.md]: Changed it moving the small part I still think is relative from and removing the rest.

Comment (Hedda) [reply org.openhab.binding.zigbee/README.md]: Changed it moving the small part I still think is relative from and removing the rest.

File: org.openhab.binding.zigbee/README.md (hunk 2 above)
Comment (Hedda) [reply org.openhab.binding.zigbee/README.md]: Changed it moving the small part I still think is relative from and removing the rest.

**Comment by Hedda** [comment]
> Please read the contribution guidelines also and please sign the commit to say that you wrote this text or have the right to it.

Can I sign that online somewhere? ...makes me wonder why it needs to be so complicated and hard to update documentation for a binding? This is documentation and not code. Are non-coders not supposed to be able to help with documentation?

**Comment by Hedda** [comment]
> > Please read the contribution guidelines also and please sign the commit to say that you wrote this text or have the right to it.
> 
> Can I sign that online somewhere? ...makes me wonder why it needs to be so complicated and hard to update documentation for a binding? This is documentation and not code. Are non-coders supposed to be able to help with documentation? I don't like Git

OMG! https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330

No wonder there are not a lot of docs contributers to openHAB.

**Comment by cdjackson** [comment]
Hey @Hedda. 

> it was I who wrote that text for that project 

Thanks - if you wrote this then as I said above that's fine - we just need to make sure that yo have ownership or approval to use the text.

> Can I sign that online somewhere?

Yes, you just need to put this one line in the commit and preferably also in the PR. It's very simple - just write the one "signed off by" line as per the contributions file -:

>     Signed-off-by: Joe Smith <joe.smith@email.com> (github: github_handle)

> No wonder there are not a lot of docs contributers to openHAB.

Well, really, it's not that hard. If you take all this time to write the docs then writing this one line is simple. Most git software provides a mechanism to do this automatically as well.

File: org.openhab.binding.zigbee/README.md [hunk 5]
```diff
@@ -153,23 +153,25 @@ The following coordinators are known to be supported.
...
 
-Firmware versions higher than 6.4.0.0 may be required to work with some newer devices (i.e. those conforming to the Zigbee 3.0 standard). If your dongle has older firmware then devices may appear to join the network, but will not work as they will leave the network if they do not receive updated security information.
+Firmware versions higher than 6.4.0.0 may be required to work with some newer devices (i.e. those conforming to the Zigbee 3.0 standard). If your dongle has older firmware then devices may appear to join the network, but will not work as they will leave the network if they do not receive updated security information. 
+
+EmberZNet Serial Protocol version 8 (EZSP v8) introduced in EmberZNet 6.7.x.x uses a new framing format and expanded command id field from 8 bits to 16 bits and is therefor not backward compatible. If you for whatever version want a backwards compatible firmware then use EmberZNet 6.6.x.x as it supports EZSP v5, v6 and v7 (so not EZSP v8).
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/README.md:174]: Please remove this line - it's not relevant for users what version of EZSP is used. This is user documentation and I would not recommend that people use old firmware.

**Review by cdjackson — CHANGES_REQUESTED** [review]

**Comment by Hedda** [comment]
Closing since apparently not possible to add signed-off to existing commits via GitHub web interface.

https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330

Maybe I create a new PR later. No promises as DCO signing method totally killed my mood for now. Posted complain here:

https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330

**Comment by Hedda** [comment]
Nope, apparently, DCO is still needed on existing/old PRs. closing again

**Pull request closed without merging**